examples/package.json

```diff
@@ -7,11 +7,14 @@
   "private": true,
   "dependencies": {
     "@mapbox/mapbox-gl-draw": "^1.0.2",
+    "@turf/turf": "^5.1.6",
```

Comment:
Is this package used somewhere?

Reply:
I was trying to implement a custom mode.. sorry I forgot to clean up unnecessary packages I was using